digitalpublicgoods/product-catalog-management-tool.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "Product Catalog Management Tool",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://gitlab.com/pcmt/pcmt/-/blob/master/COPYRIGHT.md"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://gitlab.com/pcmt/pcmt/-/wikis/home"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Imports and exports into CSV or excel formats"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Unknown",
```

Comment:
Unknown is not a valid response. Please request PCMT to share their privacy policy considering they collect PII data for administrators. Thanks.